src/failures/src/remote-failures-provider.spec.ts

```diff
@@ -0,0 +1,92 @@
+import {
+    orchestrator,
+    remoteProvider,
+    activateFailure,
+    deactivateFailure,
+    activateSimVarName,
+    deactivateSimVarName,
+    flushPromises,
+    identifier,
+    remoteActivateFailure,
+    remoteDeactivateFailure,
+} from './test-values';
+
+describe('RemoteFailuresProvider', () => {
+    it('is made aware after instantiation of the existing state', async () => {
+        const o = orchestrator();
+
+        await activateFailure(o);
+
+        // We rely here on the fact that Coherent triggers are instant in the mocks. In sim, this might
+        // take a frame or two to actually be correct - this is good enough for our use
+        const rfp = remoteProvider();
+
+        expect(rfp.isActive(identifier)).toBe(true);
+    });
+
+    it('is notified about an activated failure', async () => {
+        const o = orchestrator();
+        const rfp = remoteProvider();
+
+        await activateFailure(o);
+
+        expect(rfp.isActive(identifier)).toBe(true);
+    });
+
+    it('is notified about a deactivated failure', async () => {
+        const o = orchestrator();
+        const rfp = remoteProvider();
+
+        await deactivateFailure(o);
+
+        expect(rfp.isActive(identifier)).toBe(false);
```

Comment:
**suggestion**: First bring the test in a state where the failure is active in the rfp, then remove the failure and assert.